snmp/tests/test_e2e_core_profiles/test_profile__generic_entity_sensor.py

```diff
@@ -34,7 +33,23 @@ def test_e2e_profile__generic_entity_sensor(dd_agent_check):
 
     # --- TEST METRICS ---
     assert_common_metrics(aggregator, common_tags)
-    assert_extend_generic_entity_sensor(aggregator, common_tags)
+
+    tag_rows = [
+        [
+            'ent_phy_sensor_type:percent_rh',
+            'ent_phy_sensor_scale:micro',
+            'ent_phy_sensor_precision:0',
+            'ent_phy_sensor_units_display:driving driving forward acted their but',
+            'ent_physical_descr:example admin string',
+            'ent_physical_class:energy_object',
+            'ent_physical_name:console',
+            'ent_physical_serial_num:SN12345678',
+            'ent_physical_model_name:model name',
+            'ent_phy_sensor_oper_status:nonoperational',
+        ],
+    ]
+    for tag_row in tag_rows:
+        aggregator.assert_metric('snmp.entPhySensorValue', metric_type=aggregator.GAUGE, tags=common_tags + tag_row)
```

Comment:
I had to delete import `assert_extend_generic_entity_sensor` since it's asserting the metric with missing tags, and it would be painful to change all the profiles that are importing this function. So I asserted the metric right in this test with all necessary tags.